pom.xml

```diff
@@ -34,6 +34,11 @@ SOFTWARE
   <version>1.0-SNAPSHOT</version>
   <packaging>jar</packaging>
   <name>eo-math</name>
+  <properties>
+    <eolang.version>0.26.0</eolang.version>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
```

Comment:
@Graur I think these two properties are coming from the parent POM, no need to define them here